python/cuspatial/cuspatial/tests/test_geodataframe.py

```diff
@@ -320,6 +320,7 @@ def test_boolmask(gpdf, df_boolmask):
     assert_eq_geo_df(gi[df_boolmask], cugpdf_back[df_boolmask])
 
 
+@pytest.mark.xfail(reason="Flaky memory test")
```

Comment:
Let's link the issue in the description. That way if someone wonders about this, they know where to go for more info

```suggestion
@pytest.mark.xfail(reason="Flaky memory test. See https://github.com/rapidsai/cuspatial/issues/1352")
```